dnd.py

```diff
@@ -56,20 +58,35 @@
                 if role.is_everyone:
                     await client.edit_channel_permissions(newChannel, role, overwrite)
                     break
-    else:
-        #player specific commands
-        if message.content.startswith("!registerLanguage"):
-            print("check that the language role already exists, and add the appropriate role to the user")
+
+    if isPlayer:
+        # player specific commands
+        # print("check that the language role already exists, and add the appropriate role to the user")
+        if message.content.startswith("!iknow"):
+            language = message.content.replace("!iknow", "").strip()
+            thisList = ["DM", "Player", "admin"]
```

Comment:
admin is not a role added by the bot, its a server role, and shouldnt be considered

Reply:
Admin should be there so that a person cant make themselves an admin =)

Reply:
that is a user that is just for this server. It wouldnt be on other servers. so it doesnt make sense to check for it here